### ISSUE TITLE
Non ad-related traffic is blocked on amazon.com

### ISSUE THREAD

**Issue by ddavydenko** [easylist/easylist#1087]
### List the website(s) you're having issues:
https://www.amazon.com

`https://www.amazon.com/s/ref=nb_sb_noss_1?url=search-alias%3Daps&field-keywords=iphone+case`

### What happens?

EasyPrivacy filter seems to block traffic from amazon pages to its operational tracking domain: `https://fls-na.amazon.com`. Namely, there is a rule `||fls-na.amazon.com^` that blocks calls to URL `https://fls-na.amazon.com/1/batch/1/OE/`.

This traffic is not used to track any particular user behavior, it is rather traffic used to monitor operational aspects of the page and report various latencies and other related information about page and user experience with it. It is not used to target or track particular user behavior in any shape or form as it reports page metrics in aggregated form. You may notice that call to that URL is happening every several seconds to preserve browser resources.

Because of the effect of this rule, portion of amazon.com traffic is missing from operational analysis of whole ecosystem thus hiding any signals that might uncover potential user experience issues and trigger engineers to address those.

### List Subscriptions you're using:

EasyList + EasyPrivacy. EasyPrivacy has the offending rule.

### Your settings

- MacOS Sierra, v10.12.6 
- Chrome v66.0.3359.117
- Adblock Extension/version: uBlock Origin 1.16.0

**Comment by ryanbr** [comment]
tl;dr there is plenty of other url's that use the fls-na.amazon.com/... this way is probably the cleanest.

**Comment by ddavydenko** [comment]
You seem to have modified rule `-||fls-*.amazon.`, however that was not the one that is blocking particular URL I am asking to unblock.

From what uBlock dashboard tool (very nice!) tells me I can see that the URL I would like to get unblocked `https://fls-na.amazon.com/1/batch/1/OE/` is blocked by rule `||fls-na.amazon.com^` which is still there in privacy rules file after I updated it in my instance of uBlock.

If it is not feasible to completely remove `||fls-na.amazon.com^` rule in favor of the rest of rules that also work with fls-*.amazon.com domains, is it possible to modify privacy rules in some other way to achieve the desired outcome: ensure that calls to `https://fls-*.amazon.com/1/batch/1/OE/` are not blocked?

**Comment by ryanbr** [comment]
I have modified the filter to not to touch "amazon.com/1/batch/1/OE/".

**Comment by ryanbr** [comment]
Ofcourse, I can revert this change, and put it back in. I'm not too fussed either way.

**Comment by ddavydenko** [comment]
I tested behavior of amazon.com pages with uBlock and privacy filter on (EasyPrivacy) and I saw that the URL in question is still blocked. uBlock dashboard reported rule in question to be the one that is blocking it. 

I wonder if I need to update any other rules in addition to EasyPrivacy to start seeing fls-na requests not being blocked?

--
Thanks,
Denis

On Apr 24, 2018, at 7:41 PM, Fanboynz <notifications@github.com> wrote:

Ofcourse, I can revert this change, and put it back in. I'm not too fussed either way.
